purchase_advance_payment/data/ir_config_parameter.xml

```diff
@@ -1,9 +1,17 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <odoo noupdate="1">
-
     <record id="auto_post_advance_payments" model="ir.config_parameter" forcecreate="0">
         <field name="key">purchase_advance_payment.auto_post_advance_payments</field>
         <field name="value">True</field>
     </record>
-
+    <record
+        id="auto_reconcile_advance_payments"
+        model="ir.config_parameter"
+        forcecreate="0"
+    >
+        <field
+            name="key"
+        >purchase_advance_payment.auto_reconcile_advance_payments</field>
+        <field name="value">True</field>
```

Comment:
I would set it to `False` to avoid the risk of a breaking change in preexisting workflows for instances that already have this module installed.